deploy/examples/keycloak.json

```diff
@@ -5,6 +5,7 @@
     "name": "example"
   },
   "spec": {
-    "adminCredentials": ""
+    "adminCredentials": "",
+    "plugins": "keycloak-metrics-spi"
```

Comment:
I do not see a matching update to the CRD for this, is it not required? Would it be better to implement this as an array?

i.e.
```
"plugins": [ "keycloak-metrics-spi" ],
```